src/main/java/org/jenkinsci/plugins/electricflow/action/CloudBeesCDPBABuildDetails.java

```diff
@@ -13,14 +13,14 @@
 // TODO: Since we have now 2 classes that are doing pretty match the same
 @ExportedBean
 public class CloudBeesCDPBABuildDetails implements Action {
-  @Exported public String flowRuntimeId = "";
-  @Exported public String projectName = "";
-  @Exported public String releaseName = "";
-  @Exported public String flowRuntimeStateId = "";
-  @Exported public String stageName = "";
-  @Exported public String configurationName = "";
-  @Exported public CIBuildDetail.BuildTriggerSource triggerSource;
-  @Exported public CIBuildDetail.BuildAssociationType buildAssociationType;
+  @Exported private String flowRuntimeId = "";
```

Comment:
Has this been tested? I'm not sure if Stapler Python/XML APIs work with private methods. Again, if there is any doubt here, I would rather defer this cleanup to a separate PR.